Cha12 - typo add_allocation_to_read_model

adrienbrunet opened the pull request.

hjwp commented:
thanks Adrien!  how did you like this chapter?

I'd recommend "designing data-intensive applications" for a bit more in-depth on databases and read models and eventual consistency and whatnot...  amazing book.

adrienbrunet commented:
Well, really nice so far. I'm reading the appendix now ;)

I'm struggling to decide if I really need to embrace all of these approaches in my code (probably not needed right away).

I really like the journey of this book. With almost every chapter I'm amazed by the approach and its simplicity, then doubtful, then quite angry because it upsets the way I used to organized stuff and at the end, I'm left with hard choices to make now. It's been a blast. Changed the way I think about ORM, Flask, Django and architecture in general. I see micro-services with new eyes (I've seen too many distributed monoliths).

I'm adding this database book to my list, but I haven't finished "security engineering" (goaty recommendation if I remember correctly) and I've already added a bunch of books following "More required reading" section. Fun times to come!

I'll give a few weeks to these ideas to root in me, read more about DDD, databases et microservices. Then, I'll have a second read and try its code again. I'll keep you updated
